### PR TITLE
County profile

### PR DESCRIPTION
**Pull request by Luciekimotho**
1. Bootstrapping react app.
2. Added styling
3. Added pages
4. FAQ, About us & Dashboard pages done
5. Country profile in progress

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -64,4 +64,75 @@
 .sdg .col-md-2{
   padding-right: 0px;
   padding-left: 0px;
+}
```
Comment (KelvinWanyama) [inline src/App.css]: Could we move the CSS to independent files i.e Homepage.css Dashboard.css?

File: src/views/CountryProfile.js [hunk 2]
```diff
@@ -1,12 +1,98 @@
...
+    Container
+} from "reactstrap";
+
+
+import Highcharts from "highcharts";
+import HighchartsReact from "highcharts-react-official";
+import highchartsMap from "highcharts/modules/map";
+import africaMapData from "@highcharts/map-collection/custom/africa.geo.json";
+
+var data = require('../assets/data/trial.json');
```
Comment (KelvinWanyama) [inline src/views/CountryProfile.js]: Consider using **let**. You can read more [here](https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e)

File: src/views/CountryProfile.js [hunk 3]
```diff
@@ -74,24 +87,59 @@ const mapOptions = {
     }]
   }
 
-
+  function loadCountryData(countryId){
+    //alert(countryId);
+    var countryData = require('../assets/data/countryProfile.json');
```
Comment (KelvinWanyama) [inline src/views/CountryProfile.js]: Consider using let. You can read more [here](https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e)

File: src/App.css [hunk 4]
```diff
@@ -11,6 +11,36 @@
   background: url("../src/assets/img/bg/trevor.jpg");
 }
 
+/* Country profile page */
```
Comment (KelvinWanyama) [inline src/App.css]: Please consider moving the css to CountryProfile.css

File: src/visualizations/gauge.js [hunk 5]
```diff
@@ -6,62 +6,123 @@ import am4themes_animated from "@amcharts/amcharts4/themes/animated";
...
+        // axis.strictMinMax = true;
+        // axis.renderer.labels.template.radius = 85;
+        // axis.renderer.inside = true;
+        // axis.valueInterval = 1;
+        // axis.renderer.labels.template.disabled = true;
+        // axis.renderer.ticks.template.disabled = true;
+        // axis.renderer.grid.template.disabled = true;
+
+
+        var axis = chart.xAxes.push(new am4charts.ValueAxis());
```
Comment (KelvinWanyama) [inline src/visualizations/gauge.js]: Consider using **let** instead of **var**.

File: src/visualizations/gauge.js [hunk 6]
```diff
@@ -6,62 +6,123 @@ import am4themes_animated from "@amcharts/amcharts4/themes/animated";
...
+        axis.renderer.ticks.template.length = 10;
+        axis.renderer.labels.template.disabled = true;
+        axis.renderer.ticks.template.disabled = true;
+        axis.renderer.grid.template.disabled = true;
+        axis.renderer.labels.template.radius = 40;
+        /**
+         * Axis for ranges
+         */
+        var colorSet = new am4core.ColorSet();
+        var axis2 = chart.xAxes.push(new am4charts.ValueAxis());
```
Comment (KelvinWanyama) [inline src/visualizations/gauge.js]: Consider using **let** instead of **var**.

File: src/visualizations/gauge.js [hunk 7]
```diff
@@ -6,62 +6,123 @@ import am4themes_animated from "@amcharts/amcharts4/themes/animated";
...
+
+        // Add hand
+        let hand = chart.hands.push(new am4charts.ClockHand());
+        hand.startWidth = 10;
+        hand.radius = am4core.percent(85);
+        hand.innerRadius = am4core.percent(20);
+        hand.pin.disabled = true;
+        hand.value = 0;
+
+        var label = chart.radarContainer.createChild(am4core.Label);
```
Comment (KelvinWanyama) [inline src/visualizations/gauge.js]: Consider using **let** instead of **var**.

File: src/visualizations/gauge.js [hunk 8]
```diff
@@ -6,62 +6,123 @@ import am4themes_animated from "@amcharts/amcharts4/themes/animated";
...
+        axis.renderer.ticks.template.strokeOpacity = 1;
+        axis.renderer.ticks.template.length = 10;
+        axis.renderer.labels.template.disabled = true;
+        axis.renderer.ticks.template.disabled = true;
+        axis.renderer.grid.template.disabled = true;
+        axis.renderer.labels.template.radius = 40;
+        /**
+         * Axis for ranges
+         */
+        var colorSet = new am4core.ColorSet();
```
Comment (KelvinWanyama) [inline src/visualizations/gauge.js]: Consider using **let** instead of **var**.

File: src/App.css [hunk 9]
```diff
@@ -15,6 +15,26 @@
   padding: 0.5rem 1rem;
 }
 
+/* Individual SDG 1  */
```
Comment (KelvinWanyama) [inline src/App.css]: Consider moving this css to sdg1.css

File: src/views/sdgs/sdg.js [hunk 10]
```diff
@@ -12,7 +12,21 @@ function Sdg(){
     const  imgSrc = image(`./${sdg.image}.jpg`);
     const targets = sdg.targets;
     const mapData = require('../../assets/data/trial.json');
-   // console.log(mapData)
+   
+    var csvFile = require("../../assets/data/sdg/sdgTarget_11_gdb.csv");
```
Comment (KelvinWanyama) [inline src/views/sdgs/sdg.js]: Consider using **let/const .**

**Comment by Luciekimotho** [comment]
I've replaced var with let/const as suggested. 
For the CSS moving to the independent pages, I can work on that later

**Pull request merged**